wallet/chainntfns.go

```diff
@@ -645,15 +648,20 @@ func (w *Wallet) addRelevantTx(rec *wtxmgr.TxRecord,
 					return err
 				}
 
-				// TODO make this work, the type conversion is broken cj
-				//scrAddr, err := w.Manager.Address(addr)
-				//if err == nil {
-				//	addrTyped := scrAddr.(*waddrmgr.ManagedScriptAddress)
-				//	retrievedScript, err := addrTyped.Script()
-				//	if err == nil {
-				//		expandedScript = retrievedScript
-				//	}
-				//}
+				if expandedScript == nil {
+					scrAddr, err := w.Manager.Address(addr)
+					if err == nil {
+						sa, ok := scrAddr.(waddrmgr.ManagedScriptAddress)
+						if !ok {
+							log.Warnf("address %v is not a script"+
+								" address", sa.Address().EncodeAddress())
+						}
+						retrievedScript, err := sa.Script()
+						if err == nil {
```

Comment:
please add comment on why an error is of no concern.  Also, I'd recommend at least log.Debugf'ing the error.